pyproject.toml

```diff
@@ -29,6 +31,8 @@ mkdocs-mermaid2-plugin = "^0.6.0"
 oaklib = "^0.5.28" # https://pypi.org/project/oaklib/0.5.6/
 rdflib = "^6.2.0"
 sheets-and-friends = "^0.5.4"
+jupyter = "^1.1.1"
```

Comment:
I like these down in dev deps, is there any chance the ones above can go down here too?

Reply:
yes! my slip-up.